lib.typ

```diff
@@ -209,7 +233,7 @@
   )
   cv-line(
     date,
-    elements.join(", "),
+    elements.slice(0, -1).join(", ") + text(size: 10.5pt, elements.at(-1)),
```

Comment:
i agree with @Orbion-J on the size. something like `0.85em` or `0.9em` would probably fit better here